notes/transcript15.md

```diff
@@ -213,108 +213,228 @@ Igor: I guess the signature won't signatures won't match but we can do it a shot
+Oskar: There was another point if it was a good point it was brought up which is the lack of engagement. Just process speaking I think that partitioning topics is like a great way to optimize the bad way from the second. It is a reasonable thing to do and it seems like something that generally could be done local sort of local form decision-making within the Swarm. I think that concern might be when it comes to something that breaks the protocol so that it impacts sort of more groups of people and I think if that is made more clear or if we sort of we want to separate out all of this and have it be a sort of different cultural process and different technical process where we're changing the protocol and we need some form of consensus perhaps in these calls, perhaps in some other form. That's something we're going to work on over the next few weeks and months to sort of we have some haphazard documentation but bring it more together and also create this specs repository and then maybe we when we have questions like these, then we can bring them up in the core dev call as of come prepare with a proposal for changing some, the protocol in some way and then we can discuss it. I think that would separate out the things there are sort of swarm implementation specifics for certain trade-offs versus things that impact the entire protocol. And if people have any thoughts on that.
+
+Igor: I think that's a good idea in general. So we should probably do some some sort of list.
+
+### 45:00 - 50:00
+
+Eric: I I don't have a thought but I'm wondering so how do we make this decision whether we accept break breaking protocol or not? How do we consider..
+
+Oskar: Yes so we'll have to figure out, I think that for those who been here for it for like a year or year n a half, we had the Swarm process and it's something like this but in a way like similar to the hyphen improvement process or bittorrent improvement or Bitcoin or EIPs or whatever and it says they're having a process like that where there's some form of consensus among some sort of core contributers or whatever it looks like and that will also be its own sort of SIP or whatever you want to call it. Atleast that's how I'm thinking about right now and I think it is a pretty scalable and and and tested process for protocol changes in general and something that's generally used. So that's how I imagined. Specifically how decision will be made or when something will move from draft is always something we can discuss further.
```

Comment:
```suggestion
Oskar: Yes so we'll have to figure out, I think that for those who been here for it for like a year or year n a half, we had the Swarm process and it's something like this but in a way like similar to the hyphen improvement process or bittorrent improvement or Bitcoin or EIPs or whatever and it says they're having a process like that where there's some form of consensus among some sort of core contributers or whatever it looks like and that will also be its own sort of SIP or whatever you want to call it. Atleast that's how I'm thinking about right now and I think it is a pretty scalable and tested process for protocol changes in general and something that's generally used. So that's how I imagined. Specifically how decisions will be made or when something will move from draft is always something we can discuss further.
```